src/dialog/dialog.service.ts

```diff
@@ -77,7 +81,14 @@ export class DialogService {
 		protected componentFactoryResolver: ComponentFactoryResolver,
 		protected injector: Injector,
 		protected placeholderService: PlaceholderService
-	) {}
+	) {
+		// Adds current close subscription to the reference of all close subscriptions for
+		// local dialog service.
+		this.dialogSubscription.add(this.closeSubscription);
```

Comment:
Sorry, my bad for not being clearer, `DialogService.dialogCloseSubscription.add` should absolutely be in the constructor, but `this.dialogSubscription.add` still needs to be in `open` 👍 otherwise it won't actually track it